docs/examples/tiledata/color.js

```diff
@@ -0,0 +1,57 @@
+// @flow strict
+import { type Node, useState } from 'react';
+import { TileData, Flex } from 'gestalt';
+
+export default function Example(): Node {
+  const [selectedTileId, setSelectedTileId] = useState('1');
+
+  const isSelected = (id: string) => selectedTileId === id;
+  const handleSelection = (ev) => {
+    setSelectedTileId(ev.id);
+  };
+
+  return (
+    <Flex gap={4} wrap justifyContent="center">
+      <TileData
```

Comment:
Is there any reason we're showing just these five colors? Should we not show all available colors, either at-a-glance or using a SelectList?